src/ipc.c

```diff
@@ -0,0 +1,314 @@
+/**
+ * collectd - src/ipc.c, based on src/memcached.c
+ * Copyright (C) 2010       Andres J. Diaz <ajdiaz@connectical.com>
+ * Copyright (C) 2010       Manuel L. Sanmartin <manuel.luis@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
```

Comment:
Most of the new plugins use the MIT licence. Would you consider switching to follow the trend ?
You're of course totally free to stick to the GPL if you care about the implications of this.